pyteal/ast/opup.py

```diff
@@ -29,8 +33,23 @@ class OpUpMode(Enum):
     OnCall = 1
 
 
-ON_CALL_APP = Bytes("base16", "068101")  # v6 program "int 1"
-MIN_TXN_FEE = Int(1000)
+class OpUpFeeSource(Enum):
+    # Only the excess fee on the outer group should be used (set inner_tx.fee=0)
+    GroupExcess = 0
+    # The app's account will cover all fees (set inner_tx.fee=Global.min_tx_fee())
+    AppAccount = 1
+    # First the excess will be used, remaining fees will be taken from the app account
+    Any = 2
+
+
+def fee_by_source(source: OpUpFeeSource) -> Optional[Expr]:
```

Comment:
@barnjamin Optional:  To minimize public API surface area, do you feel comfortable marking `fee_by_source` private by renaming to `_fee_by_source`?

Feel welcomed to resolve as is.

Reply:
will do

Reply:
Resolving - Thanks for brining in the change.